react-app/src/components/projects/ProjectsTable.tsx

```diff
@@ -88,9 +88,14 @@ const ProjectsTable = () => {
       case 'All Projects':
         ref.current.setFilterModel({ items: [] });
         break;
+      case 'Approved for SPL':
+      case 'Contract in Place':
+      case 'On Market':
+      case 'Pre-Marketing':
+        ref.current.setFilterModel({ items: [{ value, operator: 'contains', field: 'Status' }] });
+        break;
```

Comment:
The way these cases are handled looks identical to what I see below. We could just remove this and let it default to the lowest case.
```suggestion
      case 'Approved for SPL':
      case 'Contract in Place':
      case 'On Market':
      case 'Pre-Marketing':
```